scripts/exgfs_wave_prep.sh

```diff
@@ -220,26 +232,27 @@
        echo "ABNORMAL EXIT: NO FILE ${file}"
        echo ' '
        set_trace
-       err=4;export err;${errchk}
+       export err=4
+       err_chk
```

Comment:
Compress this if-block and add fatal error message to `err_chk`.

Reply:
Compressed.